app/views/foreman_ansible/ansible_roles/_select.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= multiple_selects(f, :ansible_roles, ansible_roles, object.all_ansible_roles.map(&:id),
```

Comment:
I think this is where tests are failing. On a new host, `object` is nil. Call `f.object` instead of `object` and it'd be fixed